src/views/blocks/rsvp/form/form.php

```diff
@@ -33,18 +40,18 @@
 
 	<div class="tribe-left">
 		<?php if ( ! $must_login ) : ?>
-			<?php $this->template( 'blocks/rsvp/form/quantity', array( 'ticket' => $ticket ) ); ?>
+			<?php $this->template( 'blocks/rsvp/form/quantity', [ 'ticket' => $ticket ] ); ?>
 		<?php endif; ?>
 	</div>
 
 	<div class="tribe-right">
 		<?php $this->template( 'blocks/rsvp/form/error' ); ?>
 
 		<?php if ( $must_login ) : ?>
-			<?php $this->template( 'blocks/rsvp/form/submit-login', array( 'event_id' => $event_id, 'going' => $going, 'ticket_id' => $ticket_id ) ); ?>
+			<?php $this->template( 'blocks/rsvp/form/submit-login', [ 'event_id' => $event_id, 'going' => $going, 'ticket_id' => $ticket_id ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 4f2b8dc91b8de854a1042fa1c32272cd</sub>